lib/snapshot.js

```diff
@@ -1,5 +1,6 @@
 
 'use strict';
+const analyze_url = require( "./url-analyzer" );
```

Comment:
Just in terms of code style, in calypso we put internal exports after internal ones, we haven't strictly followed that in this codebase, but maybe we should 🤷 

I also wonder if this should be prefixed with an underscore i.e. `_analyze_url`